src/components/editor/NumberOrDefaultField.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { CustomField, FieldLabel, AutoField } from "@measured/puck";
+
+export type NumberOrDefault = number | "default";
+
+export type NumberFieldWithDefaultOptionProps = {
+  label: string;
+  defaultCustomValue: number;
```

Comment:
why defaultCustomValue and not default?

Reply:
This field is supposed to be the value that is in the number input by default when you toggle to custom mode.
default/defaultValue seemed confusing since "default" represents the other mode where it just uses the value from the theme.
But yeah I found it tricky to name a lot of these variables names -- open to suggestions if you have them

Reply:
If we want two separate defaults I think `default` and `defaultCustomValue` makes sense. Feel free to get Aaron's opinion on it.

Reply:
Checked with Aaron, he said this is good for now and hopefully we'll get some user feedback soon-ish and can adjust as needed